### PR TITLE
docs(sample): normalize firestore sample's region tags

### LINKED ISSUE #169: normalize firestore sample's region tags
Make sample's region tags consistent across all the languages.

### PR DESCRIPTION
**Pull request by laljikanjareeya**
Fixes #169

### CODE REVIEW

**Comment by samtstern** [comment]
@laljikanjareeya is there a reason for this change?  #169 says "make region tags consistent across all the languages" but from what I can tell these are not at all consistent with the other snippets (Android, iOS, Web, etc) and these are extremely verbose names.

Do you have some evidence of changes on other platforms to match this or maybe a plan I'm not aware of?

**Comment by laljikanjareeya** [comment]
@samtstern Currently We are working on four language i.e. NodeJs, Java, Python and Go.

> Do you have some evidence of changes on other platforms to match this or maybe a plan I'm not aware of?

I think @crwilcox is right person to answer this, @crwilcox can you share that document internally.

**Comment by crwilcox** [comment]
@samtstern I can reach out. doing this for the mentioned samples (android, ios, web) is optional, but it makes sense that we align here.

File: firestore/main/index.js [hunk 1]
```diff
@@ -63,17 +63,17 @@ async function demoInitialize(db) {
 // ============================================================================
 
 async function quickstartAddData(db) {
-  // [START add_lovelace]
+  // [START firestore_setup_dataset_pt1]
```
Comment (crwilcox) [inline firestore/main/index.js:66]: @laljikanjareeya we need to keep both tags for now. I know this is a bit verbose, but ensures we don't lose any if we try to generate docs between this change and updates to our docsite. 

Could you make these read like

```
// [START add_lovelace]
// [START firestore_setup_dataset_pt1]


/* SAMPLE CODE */

// [END firestore_setup_dataset_pt1]
// [END add_lovelace]

**Review by crwilcox — CHANGES_REQUESTED** [review]

File: firestore/main/index.js (hunk 1 above)
Comment (laljikanjareeya) [reply firestore/main/index.js:67]: Done!!

**Review by crwilcox — APPROVED** [review]

**Pull request merged**